flyteplugins/go/tasks/plugins/k8s/ray/ray.go

```diff
@@ -555,11 +556,24 @@ func getEventInfoForRayJob(logConfig logs.LogConfig, pluginContext k8s.PluginCon
 
 	var taskLogs []*core.TaskLog
 
+	// We use the creation timestamp of the RayJob as a proxy for the start time of the pods
+	startTime := rayJob.ObjectMeta.CreationTimestamp.Time.Unix()
+	// Don't have a good mechanism for this yet, but approximating with time.Now for now as is done in other plugins
+	finishTime := time.Now().Unix()
+	RFC3999StartTime := time.Unix(startTime, 0).Format(time.RFC3339)
+	RFC3999FinishTime := time.Unix(finishTime, 0).Format(time.RFC3339)
+
 	taskExecID := pluginContext.TaskExecutionMetadata().GetTaskExecutionID()
+	// Use PodRFC3339StartTime and PodUnixStartTime variable names with the term "Pod"
+	// for compatibility with existing log templates and documentation around task logs.
```

Comment:
Referring to [this](https://docs-legacy.flyte.org/en/v1.13.3/user_guide/productionizing/configuring_logging_links_in_the_ui.html#how-to-configure) docs page.